TykTechnologies/tyk-operator
# Add security policies for Tyk OSS

## Description
This PR adds support for Security Policies API in CE mode. This API was introduced in Tyk GW version 4.1.
With this change, users will be able to manage Policy objects in CE mode.
## Related Issue




Resolves #357 
https://tyktech.atlassian.net/browse/TT-3677
## Motivation and Context


## Test Coverage For This Change



1. Deploy CE testing environment
```bash
make boot-ce IMG=tykio/tyk-operator:test
```
2. Create an ApiDefinition CR
```bash
kubectl apply -f ./config/samples/httpbin_protected.yaml
```
3. Create a SecurityPolicy that will access ApiDefinition created above
```bash
kubectl apply -f ./config/samples/httpbin_protected_policy.yaml
```
4. Do CRUD operations on Security Policy and verify corresponding changes on resources.

## Screenshots (if appropriate)

## Types of changes


## Checklist
